packages/react-components/src/components/styles/SendBox.styles.ts

```diff
@@ -41,7 +41,7 @@ export const suppressIconStyle = {
 };
 
 export const sendBoxStyle = mergeStyles({
-  minHeight: '0',
+  minHeight: '2.25rem', // prevents the input text box from being sized to 0px when the meeting composite chat pane is closed.
```

Comment:
So this number won't back to what it is when the display  = show?

Reply:
This is `minHeight`. I think the idea is to avoid going down to 0 for hidden component so that internal text isn't lost.

When it's shown again, the flex container will fill up the given space (so larger than 2.25 rem).

Is my understanding correct, @JamesBurnside ?

Reply:
This is to avoid (what looks like) an edge case fluent bug, when we set display to `none` on the side pane, when fluent does its calculations for the textbox height (because the textbox has `autoSizeHeight = true` to resize when there are multiple lines) it sets the height to 0px explicitly causing this:
![image](https://user-images.githubusercontent.com/2684369/128939945-a14c1935-5fc2-440b-bcdb-64682518c2a1.png)

By setting min-height here we subvert that issue (note the textbox still has a height of 0px being applied to it but the min-height makes that redundant). 

In the collapsed scenario when you start typing in the 0px it springs up the correct size implying its definitely a fluent bug